src/lib.rs

```diff
@@ -36,39 +36,38 @@
 //! only a fixed amount of memory. In this case it only needs a pointer to the
 //! first element.
 //!
-//! The list container itself has a fixed size of a single pointer, independent
+//! The list container itself has a fixed size of a single pointer independent
 //! of the number of stored elements.
 //!
-//! Intrusive lists are often used low-level code like in operating system kernels.
-//! E.g. they can be used for storing elements that represent threads that are
-//! blocked and waiting on queue.
-//! In that case the stored elements can be on the call-stack of the
-//! caller of each blocked thread, since the call-stack won't change as long as
-//! the thread is blocked.
+//! Intrusive lists are often used in low-level code like in operating system
+//! kernels.  E.g. they can be used for storing elements that represent threads
```

Comment:
Any specific reason you merged the lines but left 2 spaces at all those places? I think this will still not result in new-lines in normal markdown. I'm ok either way - mainly curious. Thanks for not increasing the maximum line length.

Reply:
The existing text was wrapped at approximately 81 characters, so I set Emacs' `fill-column` to that and rewrapped any sections that changed.  As you note, the lack of two newlines there means that Markdown treats the entire section as one paragraph, so when I rewrapped it, I wrapped it together.

Regarding the two spaces, Emacs' wrapping algorithm adds two spaces after a sentence [by default](https://www.gnu.org/software/emacs/manual/html_node/emacs/Fill-Commands.html).  I see now, though, that the document uses one space after a sentence consistently, so I've updated the commit to preserve that.

Reply:
Ah, you merged ahead of this spacing change.  Nevermind that then.  Markdown obviously doesn't care.